idaes/power_generation/properties/natural_gas_PR.py

```diff
@@ -62,247 +64,366 @@
+        'entr_mol_ig_comp': RPP5,
+        'parameter_data': {
+            'mw': (0.058123, pyunits.kg/pyunits.mol), # RPP5
+            'pressure_crit': (37.96e5, pyunits.Pa), # RPP5
+            'temperature_crit': (425.12, pyunits.K), # RPP5
+            'omega': 0.200, # RPP5
+            'cp_mol_ig_comp_coeff': { # RPP5
+                'a0': 5.547,
+                'a1': 5.536e-3,
+                'a2': 8.057e-5,
+                'a3': -10.571e-8,
+                'a4': 4.134e-11},
+            'enth_mol_form_vap_comp_ref': (
+                -125790, pyunits.J/pyunits.mol), # RPP5
+            'entr_mol_form_vap_comp_ref': (
+                310.23, pyunits.J/pyunits.mol/pyunits.K) # wikipedia data page
+        }
+    },
+    'H2S': {
+        'type': Component,
+        'valid_phase_types': [PhaseType.vaporPhase],
+        'elemental_composition': {'H':2, 'S':1},
+        'enth_mol_ig_comp': NIST,
+        'entr_mol_ig_comp': NIST,
+        'parameter_data': {
+            'mw': (0.034081, pyunits.kg/pyunits.mol), # NIST
+            'pressure_crit': (89.6291e5, pyunits.Pa), # NIST <- Goodwin 1983
+            'temperature_crit': (373.4, pyunits.K), # NIST <- Goodwin 1983
+            'omega': 0.090, # RPP5
+            'cp_mol_ig_comp_coeff': { # NIST <- Chase 1998
+                'A': 26.88412,
+                'B': 18.67809,
+                'C': 3.434203,
+                'D': -3.378702,
+                'E': 0.135882,
+                'F': -28.91211,
+                'G': 233.3747,
+                'H': -20.50202},
+            'enth_mol_form_vap_comp_ref': (
+                -20.50e3, pyunits.J/pyunits.mol), # NIST <- Chase 1998
+            'entr_mol_form_vap_comp_ref': (
+                205.77, pyunits.J/pyunits.mol/pyunits.K), # NIST <- Chase 1998
+        }
+    },
+    'SO2': {
+        'type': Component,
+        'valid_phase_types': [PhaseType.vaporPhase],
+        'elemental_composition': {'S':1, 'O':2},
+        'enth_mol_ig_comp': NIST,
+        'entr_mol_ig_comp': NIST,
+        'parameter_data': {
+            'mw': (0.064064, pyunits.kg/pyunits.mol), # NIST
+            'pressure_crit': (78.84e5, pyunits.Pa), # RPP5
+            'temperature_crit': (430.80, pyunits.K), # RPP5
+            'omega': 0.0, # RPP5 (Not listed)
+            'cp_mol_ig_comp_coeff': { # NIST <- Chase 1998
+                'A': 21.43049,
+                'B': 74.35094,
+                'C': -57.75217,
+                'D': 16.35534,
+                'E': 0.086731,
+                'F': -305.7688,
+                'G': 254.8872,
+                'H': -296.8422},
+            'enth_mol_form_vap_comp_ref': (
+                -296.84e3, pyunits.J/pyunits.mol), # NIST <- Chase 1998
+            'entr_mol_form_vap_comp_ref': (
+                248.21, pyunits.J/pyunits.mol/pyunits.K) # NIST <- Chase 1998
+        }
+    },
+    'C2H4': {
+        'type': Component,
+        'valid_phase_types': [PhaseType.vaporPhase],
```

Comment:
Hi @eslickj : Should we add "valid_phase_types" declaration for other components too?

Reply:
No, I don't think so.  In this case, the only thing we expect to liquefy is water.  Only adding liquid phase to water should make the problem a lot simpler.  Most of the vapor pressure curves for these things are for much lower temperature ranges than we expect to have.  @andrewlee94 may want to weigh in since he has a much better understanding of how the property package is implemented, but my feeling at least for now is that we should try to keep it simple.  I guess we'll probably have supercritical CO2, but I think that should be fine.  We can also switch to Span-Wagner once we have almost pure CO2.

Reply:
The choice is entirely up to you of what components you want to assume are condensable. Fewer components will generally be easier to solve however.